docker/README.md

```diff
@@ -0,0 +1,83 @@
+You can use Docker to easily try out or test sendgrid-python.
+
+<a name="Quickstart"></a>
+# Quickstart
+
+1. Install Docker on your machine.
+2. Run `docker run -it sendgrid/sendgrid-python`.
+
+<a name="Info"></a>
+# Info
+
+This Docker image contains
+ - `sendgrid-python` and `python-http-client`
+ - Stoplight's Prism, which lets you try out the API without actually sending email
+ - A complete setup for testing the repository or your own fork
+
+Run it in interactive mode with `-it`.
+
+You can mount repositories in the `/mnt/sendgrid-python` and `/mnt/python-http-client` directories to use them instead of the default SendGrid libraries. Read on for more info.
+
+<a name="Options"></a>
+# Options
+
+## Using an old version
+
+The easiest way to use an old version is to use an old tag.
+
+```sh-session
+$ docker run -it sendgrid/sendgrid-python:v3.6.1
+```
+
+Tags from before this Docker image was created might not exist yet. You may [manually download](#Versions) old versions in order to use them.
```

Comment:
We should link them to the release tags: https://github.com/sendgrid/sendgrid-python/releases